src/Raven.Client/Documents/Queries/QueryResultBase.cs

```diff
@@ -84,5 +84,10 @@ public abstract class QueryResultBase<TResult, TInclude>
         /// Detailed timings for various parts of a query (Lucene search, loading documents, transforming results) - if requested.
         /// </summary>
         public QueryTimings Timings { get; set; }
+        
+        /// <summary>
+        /// List of longitude & latitude document properties names in a spatial query
+        /// </summary>
+        public SpatialProperty[] SpatialProperties { get; set; }
```

Comment:
Do we need to add it to `QueryResultBase`? I don't see a similar property for time series here. How do we handle that for time series?

Reply:
moved `SpatialProperties` to `QueryResultServerSide.cs`

Reply:
I like current impl much better.